custom/src/main/res/navigation/custom_nav_graph.xml

```diff
@@ -53,6 +53,17 @@
     android:id="@+id/customSettingsFragment"
     android:name="org.kiwix.kiwixmobile.custom.settings.CustomSettingsFragment"
     android:label="CustomSettingsFragment" />
+  <fragment
+    android:id="@+id/customDownloadFragment"
+    android:name="org.kiwix.kiwixmobile.custom.download.CustomDownloadFragment"
+    android:label="CustomDownloadFragment">
+    <action
+      android:id="@+id/action_customDownload_to_customReader"
+      app:destination="@id/customReaderFragment"
```

Comment:
we should create a similar action for navigating to download and then I think this should start to come together a bit more

Reply:
[6f7d35c](https://github.com/kiwix/kiwix-android/pull/2371/commits/6f7d35c3037788e7730f0f7cd31f6725f98fbb6a)